### ISSUE TITLE
Training settings

### ISSUE THREAD

**Issue by wahr0411** [uci-cbcl/UFold#7]
Hello, I'd like to ask how many epochs did you run to reach 90.5% F1-score on RNAStralign, and on other datasets? And did you adjust the learning rate while training? How can I get the pretrained model mentioned in your paper (by using which datasets how many epochs)?

Thanks

**Comment by sperfu** [comment]
Hi there,

I believe we run for about 100 epochs to get the score. As for other datasets, we can not guarantee all the epochs are the same because sometimes the 80 + epochs may already reach a plateau so there's no need for further training, so we stopped the training if training loss did not change much. We did not change the learning rate during training. All the pretrained models are stored in our cloud drive, please check it out in the font page in  our github Readme file.

Thanks

**Comment by wahr0411** [comment]
I'd also like to ask that did you test the model and calculate the f1 score every epoch? The loss I record seems fine (going down and reaching a plateau), but the f1 score for every epoch is rather random, I don't know if this is a normal situation. And how much time did you cost for training each epoch? I run it on one GTX 3080 with batchsize=1, it cost about 2 hours to train an epoch (slower than I expect). I wonder if I did something wrong while training, because the model seems not so complex.
